deno/deepgram-transcribe-video/src/mod.ts

```diff
@@ -0,0 +1,53 @@
+export default async function (req: any, res: any) {
+  const APIkey = req.variables["DEEPGRAM_API_KEY"];
```

Comment:
best practice for variable names in deno/typescript is to be camel-cased:
```suggestion
  const apiKey = req.variables["DEEPGRAM_API_KEY"];
```